lib/buchungsstreber/parser.rb

```diff
@@ -48,10 +49,10 @@ def parse_time(time_descr)
       end
     end
 
-    def qarter_time(time)
-      quarter_hours = ((time * 60 * 60) / 900).ceil
+    def minimum_time(time, minimum_time_value)
+      minimum_time = ((time * 60 * 60) / 900).ceil
 
-      quarter_hours * 0.25
+      minimum_time * minimum_time_value
```

Comment:
Meaning here you could use `@mininum_time` from above and avoid all changes to the parsers.

Reply:
Ah, disregard me... no, you're right... it's the mixin below which actually uses it, I'll have a look on how...